news/4645-use-conda-index

```diff
@@ -0,0 +1,22 @@
+### Enhancements
+
+* Depend on standalone conda-index instead of bundled indexing code. (#4690)
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* `conda-build index --file` is no longer supported.
+* `conda-build index --hotfix-source-repo` is removed.
+* Prefer `python -m conda_index`, instead of `conda-build index` or `conda
+  index`, to use faster indexing code.
```

Comment:
I think the users should prefer the use of the index subcommand long-term, when conda-index has implemented a plugin based subcommand. In the meantime let's not send them down the `-m conda_index` rabbit hole here in the changelog, and instead point them to the conda-index docs. That'll allow us to have one source of truth about how to interact with conda-index long-term, its own docs.

```suggestion
* Prefer the [standalone conda-index package](https://conda.github.io/conda-index/),
  instead of `conda-build index` or `conda index`, to use faster indexing code.
```

Reply:
A reminder that `conda index` is currently an alias for `conda-build index`